app/modules/ads/module.js

```diff
@@ -101,62 +103,76 @@ class Ads {
   }
 
   configureBillTheLizard(instantGlobals) {
-    const { context, slotService } = window.Wikia.adEngine;
-    const { billTheLizard } = window.Wikia.adServices;
+    const { context } = window.Wikia.adEngine;
+
+    if (context.get('bidders.prebid.bidsRefreshing.enabled')) {
+      this.cheshirecatCalled = false;
+      this.cheshirecatConfig = instantGlobals.wgAdDriverBillTheLizardConfig || {};
+      this.cheshirecatPredictions = {};
+
+      context.set('bidders.prebid.bidsRefreshing.bidsBackHandler', this.callCheshireCat.bind(this));
```

Comment:
Can we make it other way around? As far as I understand we need to call cat after each incontent IF there are some changes in bids. Which means that we can set flag `shouldCallCat` on each `bidsBackHandler` and then call cat after each incontent render ended (assuming that flag has changed). What do you think?